webdriver/tests/state/text/text_handling_test.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from tests.support.inline import inline
+
```

Comment:
This is just a straight port of the selenium tests. I want them in their own files as we need to give attribution to them.